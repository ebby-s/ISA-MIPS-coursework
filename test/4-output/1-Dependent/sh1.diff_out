2d1
< TB : INFO : RAM_ACCESS: Read from 0x00000000, data: 0x00140000
