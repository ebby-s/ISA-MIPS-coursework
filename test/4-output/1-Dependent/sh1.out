TB : INFO : RAM_ACCESS: Read from 0xbfc00000, data: 0x26100008
TB : INFO : RAM_ACCESS: Write to 0x00000000, data: 0x0014xxxx
TB : INFO : register_v0=00000014
TB : Finished : active=0
